community/index.md

```diff
@@ -8,9 +8,30 @@ Popular ways to connect with the Scala community include forums, chat rooms, loc
 
 The community is also the source of many libraries, tools, and other resources around Scala.
 
+## Who's behind Scala?
+
+The Scala language and associated websites are cooperative projects of
+the [Scala Center](https://scala.epfl.ch) at [EPFL](http://epfl.ch),
+[Lightbend](https://www.lightbend.com), and the Scala community
+more broadly, with participation from many companies, organizations,
+and individuals.
+
+The Scala team at Lightbend concentrates on advancing the language,
+compiler, and standard library.  The Scala Center focuses on education
+(especially online courses), documentation, open source community
+outreach, and tooling.  Community participation in all of these
+efforts is strongly encouraged.
+
+Scala was created by
+[Prof. Martin Odersky](http://lampwww.epfl.ch/~odersky/).  Martin and
+his research group at EPFL remain heavily involved with the language,
+including developing of [Dotty](http://dotty.epfl.ch), a
+next-generation research compiler for Scala.
+
 ## Forums and Mailing Lists
 
-We are in the process of migrating from Google Groups to Discourse.
+For discussions around Scala, we are in the process of migrating from
+Google Groups to Discourse.
```

Comment:
The contributors.scala-lang.org link is intentionally postponed because newbies are likelier to want scala-user. (Eventually, users.scala-lang.org will be linked instead, but it isn't ready yet.

Or do you mean link to discourse.org, to answer the "what is Discourse?" question, since many people won't know? I think in an earlier draft I actually had linked to discourse.org, but I took it out because we want to keep the focus on Scala — we don't want someone going off on a tangent. We'd rather they just clicked on the link to contributors.scala-lang.org, which follows pretty quickly — they'll see very quickly there what Discourse is like.